ansible/post/files/approve_and_issue.sh

```diff
@@ -16,97 +16,97 @@ POWER_COUNT="${2}"
 # Var: ${self.triggers.approve}
 POWER_PREFIX="${3}"
 
-APPROVED_WORKERS=0
-ISSUED_WORKERS=0
+# Machine Prefix
+MACHINE_PREFIX="${POWER_PREFIX}-worker"
+
+# Check count for Power worker/s to be added is not zero
+if [ "0" -eq "${POWER_COUNT}" ]
+then
+  echo "Supplied Worker count is zero hence exiting."
+  exit 0
+fi
 
+# Setting values for variables
+APPROVED_WORKERS=0
```

Comment:
See L50 where it's reset

Reply:
Earlier used $ISSUED_WORKERS was always set to 0 as it was not incremented. So I removed it.
$LOCAL_WORKER_SCAN - it is getting incremented hence I have used it in code.